rest/data/interface.go

```diff
@@ -0,0 +1,14 @@
+package data
+
+import (
+	"github.com/evergreen-ci/sink/model"
+	"github.com/evergreen-ci/sink/util"
+)
+
+type Connector interface {
+	// PerformanceResult
+	FindPerformanceResultById(string) (*model.PerformanceResult, error)
+	FindPerformanceResultsByTaskId(string, util.TimeRange, ...string) ([]model.PerformanceResult, error)
+	FindPerformanceResultsByVersion(string, util.TimeRange, ...string) ([]model.PerformanceResult, error)
+	FindPerformanceResultWithChildren(string, int, ...string) ([]model.PerformanceResult, error)
```

Comment:
can these return the rest model rather than the db model?

Reply:
done